### PR TITLE
Create .gitignore file

### LINKED ISSUE #3: Missing .gitignore file
## Description
It's very common to exclude files you usually don't need to track from your git repository. Some examples are machine generated files and executables.

The way to do this is with a `.gitignore` configuration file that contains a list of files to ignore.
Please, add one to your project that excludes:

- All files ending with `.o`
- The final executable of your project (`LogInDataBase`)

Reminder: Feel free to ask any questions.

## References
https://swcarpentry.github.io/git-novice-es/06-ignore/
https://www.atlassian.com/git/tutorials/saving-changes/gitignore
https://git-scm.com/docs/gitignore

### PR DESCRIPTION
**Pull request by marcosmarp**
Closes #3

### CODE REVIEW

**Review by nahueespinosa — CHANGES_REQUESTED** [review]
Cool! One more think you will need to do in this PR is to untrack the files that git is already tracking.
After that git will ignore them with these settings.

https://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/

**Comment by marcosmarp** [comment]
\

File: main.cpp [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
-				
-			case 3:
-				return 3;
-
-       }
-
-	}
-	
-}
+#include "base_functions.h"
```
Comment (nahueespinosa) [inline main.cpp:1]: The fact that these files show up as changes might indicate something went wrong when running `git init` on an already initialized repo.

**Review by nahueespinosa — CHANGES_REQUESTED** [review]
I think the easiest solution would be to create a new branch and try again honestly.
Take a look at https://github.com/marcosmarp/signIn-signUp/issues/3#issuecomment-894684791

**Comment by marcosmarp** [comment]
Closing cause it was resolved in #8

**Pull request closed without merging**